velox/serializers/CompactRowSerializer.cpp

```diff
@@ -64,14 +72,27 @@ class CompactRowVectorSerializer : public IterativeVectorSerializer {
     buffers_.push_back(std::move(buffer));
```

Comment:
NYC:
l66 add to check totalRows is zero?
l71 s/auto rawBuffer/auto* rawBuffer/